shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -629,6 +629,9 @@ - (void)viewDidAppear:(BOOL)animated {
   [[_engine.get() lifecycleChannel] sendMessage:@"AppLifecycleState.resumed"];
 
   [super viewDidAppear:animated];
+  if (UIApplication.sharedApplication.applicationState == UIApplicationStateBackground) {
```

Comment:
Is this case, shoudn't we just not sent the `AppLifecycleState.resumed` message?

Reply:
Yeah, that's much better